app/src/main/java/me/saket/dank/notifs/MediaDownloadService.java

```diff
@@ -418,9 +435,13 @@ public void onResourceReady(Bitmap imageBitmap, Transition<? super Bitmap> trans
 
             // Taking advantage of O's tinted media notifications! I feel bad for this.
             // Let's see if anyone from Google asks me to remove this.
-            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            // ---
+            // Android 11 finally broke this so we'll force it to use BigPicture style for now
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O && Build.VERSION.SDK_INT < 30) {
```

Comment:
Can we use named version code here instead of `30`? Perhaps with `<=` if latest is not available.

Reply:
SDK 28 is used so latest version listed there is api 28 / Android 9. I think the only way to get new consts is to unlink compileSdk and targetSdk in gradle scripts and switch to compile 30 + target 28. This way it shouldn't break anything, though sdk update is entirely different matter anyway

Reply:
Ok I see, lets leave this as is then.